sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -276,13 +279,23 @@ private void createPrePopulatedDocs(int numberOfPreCreatedDocuments) {
                     PojoizedJson x =
                         resp.getItem();
                     return x;
+                }).onErrorResume(throwable -> {
+                    failureCount.incrementAndGet();
+                    logger.error("Error during pre populating item "+throwable.getMessage());
+                    return Mono.empty();
+                }).doOnSuccess(pojoizedJson -> {
+                    successCount.incrementAndGet();
                 }).flux();
                 createDocumentObservables.add(obs);
             }
             docsToRead.put(container.getId(),
                 Flux.merge(Flux.fromIterable(createDocumentObservables), 100).collectList().block());
             logger.info("Finished pre-populating {} documents for container {}",
-                numberOfPreCreatedDocuments, container.getId());
+                successCount.get()-failureCount.get(), container.getId());
```

Comment:
please fix the code style.
```suggestion
                successCount.get() - failureCount.get(), container.getId());
```

Reply:
done